test/unit/test.tone_analyzer.v3.js

```diff
@@ -68,7 +68,8 @@ describe('tone_analyzer.v3', function() {
     const options = {
       text: tone_request.text,
       tones: 'emotion',
-      sentences: true
+      sentences: true,
+      language: 'en'
```

Comment:
Please make this a new test instead of modifying the existing one. Also, please add an assert to the new test to validate that the header was actually set.